src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/ReadPosition.java

```diff
@@ -14,9 +14,18 @@
 import java.util.OptionalInt;
 
 /**
- * Median distance of variant starts from ends of reads supporting each allele.
+ * Median distance of variant starts from ends of reads supporting each alt allele.
  *
- * Created by David Benjamin on 3/20/17.
+ * </p>The output is an array containing, for each alt allele, the median distance of the variant start from the closest read end over all reads that best match that allele.</p>
+ * </p>For example, for variant context with ref allele A and alt allele C the read position for alt-supporting read GGGGCTT is 2 because the A to C
+ * substitution is 2 bases from the right end of the read, which is less than its distance from the left end.
+ * For variant context with ref allele AG and alt allele A (deletion) the read position of alt-supporting read ATTTTT is 0.
```

Comment:
This is still confusing. I thought we want to keep track of sequencer cycles as errors tend to occur in later cycles, but here you are saying this annotation is independent of sequencer cycle and the closest end is what is counted. Then at the end you give an example where instead of 0, the distance to the end is 20, which seems to imply read directionality is part of the equation. These details are up to you.

What I suggest is we use some drawing conventions I learned from a forum user and surround these by `<pre>`, e.g.:
```
0     6
|     |
GGGGCTT
    ^
    A-->C is either 4 or 2 bases from end of read. Take lower number OR use read orientation for cycle information
```
 If your examples are finalized I can draw these out for you.

Reply:
This annotation is aimed at filtering mapping errors and not sequencer errors, which is why both ends are treated the same.  Hopefully BQSR would correct for sequencer cycle before reads reach M2.  I took out mention of sequencer cycles in order to avoid confusion.  I also specified that in the last example the substitution is preceded by 80 matching bases so that it is clear the right end is the closest.  Given that, the examples are final.

Reply:
Alright, thanks for changing the last example from 20 to 0. Now it all jibes.